package.json

```diff
@@ -144,9 +144,9 @@
     "build-release": "node ./scripts/release.js"
   },
   "bin": {
-    "cucumber.js": "./bin/cucumber.js",
-    "cucumber-js": "./bin/cucumber.js",
-    "cucumberjs": "./bin/cucumber.js"
+    "cucumber.js": "node ./bin/cucumber.js",
+    "cucumber-js": "node ./bin/cucumber.js",
+    "cucumberjs": "node ./bin/cucumber.js"
```

Comment:
These should not be prefixed with node. npm creates symlinks based on these values so they need to stay as just paths